src/main/steps/task-list/utils.ts

```diff
@@ -126,3 +126,32 @@ export const getAdoptionCertificateDetailsStatus = (userCase: CaseWithId): Secti
     ? SectionStatus.NOT_STARTED
     : SectionStatus.IN_PROGRESS;
 };
+
+const addressComplete = (userCase: CaseWithId, fieldPrefix: FieldPrefix) => {
+  const address1 = userCase[`${fieldPrefix}Address1`];
+  const addressTown = userCase[`${fieldPrefix}AddressTown`];
+  const addressPostcode = userCase[`${fieldPrefix}AddressPostcode`];
+  const addressCountry = userCase[`${fieldPrefix}AddressCountry`];
+
+  return (address1 && addressTown && addressPostcode) || (address1 && addressCountry);
+};
+
+export const getOtherParentStatus = (userCase: CaseWithId): SectionStatus => {
```

Comment:
Can you also cover the unit tests for this method?

Reply:
Refer https://github.com/hmcts/adoption-web/blob/90f391557b43b8a5b813d6f46c3fd3dd29587501/src/main/steps/task-list/utils.test.ts#L93